.github/workflows/integrate.yaml

```
@@ -31,6 +31,9 @@ jobs:
       - name: Add Firebase Messaging
         run: ./scripts/add-firebase-messaging.sh
       - run: flutter analyze
+      - name: Make .env file
+        run: echo "${{ secrets.TEST_MATRIX_USERNAME }}" >> .env
+        run: echo "${{ secrets.TEST_MATRIX_PASSWORD }}" >> .env
```

actionlint (1.7.12)

[error] key "run" is duplicated in element of "steps" section. previously defined at line:35,col:9

(syntax-check)